.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -2,151 +2,254 @@
+        - [ ] If new feeds were added to NuGet.config (i.e. [there's a diff
+              in NuGet.config](https://github.com/dotnet/source-build/commit/f07ff3ab98992e460ae605f609783f3abba20601#diff-9c5952957709545aad811b400e7e516eebade1e44fc4fbc23203403ffeb92c34)),
+              copy them to [smoke-testNuGet.config](https://github.com/dotnet/source-build/blob/release/3.1/smoke-testNuGet.Config)
+              and the `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS` section in `build.sh`.
+        - [ ] Add the servicing feed to build.sh and smoke-testNuGet.config
+          1. The URL will look like this: <https://pkgs.dev.azure.com/dnceng/internal/_packaging/3.1.419-servicing.22219.28-shipping/nuget/v3/index.json>
+          1. In build.sh, add this feed to the `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS`.
+          1. In smoke-testNuGet.config, add another feed entry that looks like
+             the others, with key="5.0.408-servicing.22219.28-shipping"
+             (for example) and value as the servicing url.
+      - [ ] In `eng/Version.Details.Xml`, verify that the `<Sha>` tag for all
+            repos were updated, except for these expected to usually be stable:
+        - arcade
+        - sourcelink
+        - aspnet-xdt
+        - cliCommandLineParser
+        - xliff-tasks
+        - Newtonsoft.Json
+        - common
+        - Roslyn (except for feature band updates)
+        - MSBuild (except for feature band updates)
+        - NuGet-client (except for feature band updates)
+        - fsharp (except for feature band updates)
+        - VSTest (except for feature band updates)
+1. - [ ] [Regenerate patches as necessary](patch-updates.md).
+      - Run `./build.sh` to begin clone and patch application and see what,
+        if anything, needs to be regenerated.
+      - ASP.NET will always need to be regenerated because it contains a
+        version number that needs to be updated.  The version should be updated
+        to the SDK version being built.  See <https://github.com/dotnet/source-build/blob/release/3.1/patches/aspnetcore/0006-Fix-version-number.patch>.
+1. - [ ] [Generate a PAT for AzDo](https://dev.azure.com/dnceng/_usersSettings/tokens).
+      - This should include the `dnceng` organization.
+      - Scope should include Read for Code and Packages.
+1. - [ ] Start building locally (for quick diagnosis) and in CI (for super
+         clean build) and iterate towards a green build.
+      - For local builds:
+        - Set environment variable `internalPackageFeedPat` to the generated PAT.
+          - E.g. run `export internalPackageFeedPat;read internalPackageFeedPat`
+            and paste the PAT. This sets up your running shell for an authenticated
+            build. Passing `/p:` isn't sufficient.
+        - Run `build.sh /p:AzDoPat=$internalPackageFeedPat`.
+      - For CI builds:
+        - To run CI, from [dnceng/internal/source-build-pre-arcade-CI](https://dev.azure.com/dnceng/internal/_build?definitionId=251),
```

Comment:
Consider adding a comment to ensure all local changes are pushed to the dnceng remote.

Reply:
Added.